platform-enterprise_versioned_docs/version-25.1/enterprise/_templates/k8s/data_studios/server.yml

```
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: connect-server
+  labels:
+    app.kubernetes.io/component: connect-server
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: connect-server
+  serviceName: connect-server
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: connect-server
+    spec:
+      imagePullSecrets:
+        - name: "cr.seqera.io"
+      nodeSelector:
+        kubernetes.io/arch: amd64
+      containers:
+        - name: server
+          image: cr.seqera.io/private/nf-tower-enterprise/data-studio/connect-server:0.8
```

```suggestion
          image: cr.seqera.io/private/nf-tower-enterprise/data-studio/connect-server:0.8.0
```
Same revert needed. Sorry!